documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -362,6 +365,37 @@ ifdef::community[]
 == Confluent Schema Registry
 
 There is an alternative https://github.com/confluentinc/schema-registry[schema registry] implementation provided by Confluent.
+
+[id="overview-of-deploying-confluent-schema-registry"]
+=== Confluent Schema Registry deployment overview
+
+To install standalone Confluent Schema Registry, please see Confluent's https://docs.confluent.io/platform/current/installation/installing_cp/overview.html[official deployment guide].
+
+[NOTE]
+====
+There's also possibility to https://docs.confluent.io/platform/current/installation/docker/installation.html[install] it as a container.
+====
```

Comment:
```suggestion
As an alternative, you can https://docs.confluent.io/platform/current/installation/docker/installation.html[install] the standalone Confluent Schema Registry as a container.
```
I would add this information in a standard paragraph, rather than presenting it in a note.